common/api/summary/ecschema-metadata.exports.csv

```diff
@@ -50,7 +50,7 @@ beta;diagnosticCategoryToString(category: DiagnosticCategory): "Error" | "Warnin
 beta;DiagnosticCodes:
 beta;Diagnostics:
 beta;DiagnosticType
-beta;diagnosticTypeToString(type: DiagnosticType): "Schema" | "None" | "CustomAttributeContainer" | "Property" | "RelationshipConstraint" | "SchemaItem"
```

Comment:
Worried this may fail on the build machine...

Reply:
ugh, why does this keep getting into my changes? I didn't change anything in that package.